zowe-versions.yaml

```diff
@@ -103,6 +103,9 @@ extras:
     next: true
   zos-make-for-zowe-cli:
     next: true
+  id-federation-for-zowe-cli:
+    zowe-v1-lts: true

```

Comment:
AFAIK your plugin doesn't have a `zowe-v1-lts` tag but does have a `next` tag. So instead of `zowe-v1-lts` I think this line should be changed to `next: true`. With this configuration, our nightly job will mirror all your tags from Artifactory:
![image](https://github.com/zowe/zowe-cli-standalone-package/assets/22344007/10431e35-7fab-4755-ac2a-20b0cdc6470a)